### PR TITLE
[bz-1013502] 4 WS quickstarts added that match with code snippets in …

### PR DESCRIPTION
**Pull request by rsearls**
…6.X JBoss Development Application Platform Development Guide.

### CODE REVIEW

**Comment by sgilda** [comment]
@rsearls : Thanks, I'll review this in the next day or 2.

**Comment by sgilda** [comment]
@rsearls : For EAP 7 quickstarts, we now add the JBoss EAP Maven repositories to the quickstart POM files. Users will no longer need to configure the Maven settings. If you look at the other quickstarts, you'll see they now include repositories. Would you mind updating the POM files with this info?

Also, I noticed that some of the quickstarts use these repositories: 
- http://onejar-maven-plugin.googlecode.com/svn/mavenrepo : this is not allowed. It has to use 3rd party software in Maven Central
- http://repository.jboss.org/nexus/content/groups/public/ : This is okay for development only, but must be in in maven.repository.redhat.com for product builds. Will this be done in time for the next EAP 7 DR build?

@rafabene : Please let me know if I provided the correct information.

**Comment by sgilda** [comment]
@rsearls : I issued this pull back to your branch with fixes for EAP 7: https://github.com/rsearls/jboss-eap-quickstarts/pull/1

Please review and merge if it looks okay to you.

File: jaxws-addressing/pom.xml [hunk 1]
```diff
@@ -0,0 +1,151 @@
...
+   </properties>
+
+   <modules>
+      <module>service</module>
+      <module>client</module>
+   </modules>
+
+   <dependencyManagement>
+      <dependencies>
+         <!-- Define the version of JBoss' Java EE 6 APIs we want to import. Any
```
Comment (rafabene) [inline jaxws-addressing/pom.xml:118]: We should change every Java EE 6 occurrence to Java EE 7

File: jaxws-addressing/service/pom.xml [hunk 2]
```diff
@@ -0,0 +1,64 @@
...
+-->
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/maven-v4_0_0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+
+   <parent>
+      <groupId>org.jboss.quickstarts.eap</groupId>
+      <artifactId>jboss-jaxws-addressing</artifactId>
+      <version>6.4.0-redhat-SNAPSHOT</version>
```
Comment (rafabene) [inline jaxws-addressing/service/pom.xml]: version=7.0.0-SNAPSHOT

File: jaxws-addressing/client/pom.xml [hunk 3]
```diff
@@ -0,0 +1,74 @@
...
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0
+                             http://maven.apache.org/maven-v4_0_0.xsd">
+   <modelVersion>4.0.0</modelVersion>
+
+   <parent>
+      <groupId>org.jboss.quickstarts.eap</groupId>
+      <artifactId>jboss-jaxws-addressing</artifactId>
+      <version>6.4.0-redhat-SNAPSHOT</version>
```
Comment (rafabene) [inline jaxws-addressing/client/pom.xml]: version=7.0.0-SNAPSHOT

**Comment by rafabene** [comment]
Since this PR is targeting to 7.0.x-develp branch, I made some comments regarding the EE and EAP version, but I'm starting to feel that this PR should be targeted to 6.0.x branch , am I wrong ?

**Comment by sgilda** [comment]
@rafabene : @rsearls created these for use by the documentation and at this point they can only be added for JBoss EAP 7. We are not adding new quickstarts to the 6.0.x branch at this point. We have no plans to release the EAP 6 quickstarts and, even if we did, they would have to go through the whole QE process and we don't have resources for that.

I will make the suggested updates and merge.

**Comment by sgilda** [comment]
@rafabene and @rsearls : I made the Java EE 6 and other jboss-6 changes and merged. If you see any other issues, please let me know.

I just realized I did not add these quickstarts to the root POM. I will issue a separate pull for that.

**Pull request closed without merging**